main.sh

```diff
@@ -4,17 +4,17 @@ CURRENT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 [[ -z "$TMUX_FZF_ORDER" ]] && TMUX_FZF_ORDER="copy-mode|session|window|pane|command|keybinding|clipboard|process"
 source "$CURRENT_DIR/scripts/.envs"
 
+items="$(echo $TMUX_FZF_ORDER | tr '|' '\n')"
+
 # remove copy-mode from options if we aren't in copy-mode
-if [[ "$TMUX_FZF_ORDER" == *"copy-mode"* ]] && [ "$(tmux display-message -p '#{pane_in_mode}')" -eq 0 ]; then
```

Comment:
So the code assumes `$TMUX_FZF_ORDER` is exactly `"copy-mode|session|window|pane|command|keybinding|clipboard|process"`, but in fact users can customize `$TMUX_FZF_ORDER`, as described in README. That's why there is a detection:

```shell
[[ -z "$TMUX_FZF_ORDER" ]] && TMUX_FZF_ORDER="copy-mode|session|window|pane|command|keybinding|clipboard|process"
```

The if condition `[ "$TMUX_FZF_ORDER" == *"copy-mode"* ]` is used to detect if there is a substring `"copy-mode"` in `$TMUX_FZF_ORDER`, so it should not be removed, although the functionality didn't change.

Reply:
I removed it because it makes the code simpler and the functionality didn't change. With this pr, if the substring `copy-mode` isn't present in `$TMUX_FZF_ORDR` sed won't remove anything